src/hotspot/share/runtime/frame.hpp

```diff
@@ -480,6 +480,9 @@ class frame {
   // Usage:
   // assert(frame::verify_return_pc(return_address), "must be a return pc");
 
+  // Additional interface for interpreter frames:
+  static int interpreter_frame_monitor_size_in_bytes();
```

Comment:
Please move this, without comment, to after `interpreter_frame_monitor_size` (L372)